dev/TEAL.md

```diff
@@ -646,6 +644,11 @@ Global fields are fields that are common to all the transactions in the group. I
 | 15 | AssetCreateMinBalance | uint64 | v10  | The additional minimum balance required to create (and opt-in to) an asset. |
 | 16 | AssetOptInMinBalance | uint64 | v10  | The additional minimum balance required to opt-in to an asset. |
 | 17 | GenesisHash | [32]byte | v10  | The Genesis Hash for the network. |
+| 18 | PayoutsEnabled | bool | v11  | Whether block proposal payouts are enabled. |
+| 19 | PayoutsGoOnlineFee | uint64 | v11  | The fee required in a keyreg transaction to make an account incentive eligible. |
```

Comment:
Should we specify this is in `microalgos`?

Reply:
Looking a couple lines up to `AssetCreateMinBalance` and friends, it seems we don't always bother to say that, so I'm inclined to keep these short and consistent with that style.

Reply:
Addressed in #107